index.js

```diff
@@ -1,30 +1,44 @@
 const score=document.querySelector(".score")
-let timeLeft=document.querySelector(".time-left")
+const timeLeft=document.querySelector(".time-left")
```

Comment:
Minor changes like swapping let for const do not have direct impact on the implement of hitting mole. So such changes could have been left for later when we r doing refactoring.

Reply:
sure